recipes/repoze.who-friendlyform/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "repoze.who-friendlyform" %}
+{% set version = "1.0.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 77138a94b11e8f846889fd7e025d6914e5966125869f6d75ef216801e29cdf13
+  patches:
+    - patch3.patch
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+
+  run:
+    - python
+    - setuptools
```

Comment:
Just to double-check.. Do you really need ``setuptools`` to run this project?

Reply:
setup tools removed.